robot/impl/jobmanager_test.go

```diff
@@ -0,0 +1,1291 @@
+				Name:  "switch",
+				API:   sw.API,
+			},
+		},
+		Jobs: []config.JobConfig{
+			{
+				config.JobConfigData{
+					Name:     "arm job",
+					Schedule: "3s",
+					Resource: "arm",
+					Method:   "GetGeometries",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "audio input job",
+					Schedule: "3s",
+					Resource: "audio",
+					Method:   "Properties",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "base job",
+					Schedule: "3s",
+					Resource: "base",
+					Method:   "IsMoving",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "board job",
+					Schedule: "3s",
+					Resource: "board",
+					Method:   "GetGPIO",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "button job",
+					Schedule: "3s",
+					Resource: "button",
+					Method:   "Push",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "camera job",
+					Schedule: "3s",
+					Resource: "camera",
+					Method:   "GetProperties",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "encoder job",
+					Schedule: "3s",
+					Resource: "encoder",
+					Method:   "GetProperties",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "gantry job",
+					Schedule: "3s",
+					Resource: "gantry",
+					Method:   "Home",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "generic job",
+					Schedule: "3s",
+					Resource: "generic",
+					Method:   "DoCommand",
+					Command: map[string]any{
+						"any": "any",
+					},
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "gripper job",
+					Schedule: "3s",
+					Resource: "gripper",
+					Method:   "Grab",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "input job",
+					Schedule: "3s",
+					Resource: "my_input",
+					Method:   "GetEvents",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "motor job",
+					Schedule: "3s",
+					Resource: "motor",
+					Method:   "SetPower",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "movement sensor job",
+					Schedule: "3s",
+					Resource: "mov_sensor",
+					Method:   "GetOrientation",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "pose job",
+					Schedule: "3s",
+					Resource: "pose",
+					Method:   "GetPoses",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "power sensor job",
+					Schedule: "3s",
+					Resource: "power_sensor",
+					Method:   "GetVoltage",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "sensor job",
+					Schedule: "3s",
+					Resource: "sensor",
+					Method:   "GetReadings",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "servo job",
+					Schedule: "3s",
+					Resource: "servo",
+					Method:   "GetPosition",
+				},
+			},
+			{
+				config.JobConfigData{
+					Name:     "switch job",
+					Schedule: "3s",
+					Resource: "switch",
+					Method:   "GetNumberOfPositions",
+				},
+			},
+		},
+	}
+	defer func() {
```

Comment:
Per my previous comment, I think this block and the analogous one in the services test of deferred deregistrations is fine to keep given that it saves quite a few lines that are just `}()`.